scripts/in_container/verify_providers.py

```diff
@@ -826,6 +830,12 @@ def run_provider_discovery():
     subprocess.run(["airflow", "providers", "secrets"], check=True)
     console.print("[bright_blue]List all auth backends[/]\n")
     subprocess.run(["airflow", "providers", "auth"], check=True)
+    if packaging.version.parse(airflow.version.version) > packaging.version.parse("2.6.0"):
```

Comment:
```suggestion
    if packaging.version.parse(airflow.version.version) >= packaging.version.parse("2.6.0"):
```

Reply:
I think that's more accurate

Reply:
(It's not affecting out tests now - but when in the future we will run USE_AIRFLOW_VERSION="2.6.0", the trigger **could** be running but with `>` it would have been excluded.

Reply:
I think this command we have not released yet. PR where we added it https://github.com/apache/airflow/pull/30822

Reply:
Ah right! my mistake, ... Then  I believe the best one i `>=2.7.0.dev0`. There are some weird rules in packaging but generally speaking  `dev < a < b < rc  < FINAL VERSION` so `>=2.7.0

Reply:
so it would release either in 2.6.1 or 2.7.0

Reply:
We won't release it in 2.6.1 -> it's a new feature :)

Reply:
yes, that makes sense

Reply:
Simply `2.6.1+` is bugfix only